packages/cloudflare/src/api/get-cloudflare-context.ts

```diff
@@ -71,7 +71,10 @@ async function getCloudflareContextInNextDev<
     const { getPlatformProxy } = await import(
       /* webpackIgnore: true */ `${"__wrangler".replaceAll("_", "")}`
     );
-    const { env, cf, ctx } = await getPlatformProxy();
+    const { env, cf, ctx } = await getPlatformProxy({
+      // This allows the selection of a wrangler environment while running in next dev mode
+      environment: process.env.CF_WRANGLER_ENV,
```

Comment:
```suggestion
      environment: process.env.NEXT_DEV_WRANGLER_ENV,
```

Unfortunately `.env` files are also loaded by `wrangler` (we have a collision here) so using a prefix specific to NEXT can help (See existing [env vars](https://github.com/cloudflare/workers-sdk/blob/f6132761c80d90c3521c93d48d0f0aed62bb360a/packages/wrangler/src/environment-variables/factory.ts#L1) used by wrangler)

Reply:
Makes sense - updated